icu4c/source/i18n/unicode/unumberformatter.h

```diff
@@ -584,6 +594,11 @@ unumf_openForSkeletonAndLocaleWithError(
        const UChar* skeleton, int32_t skeletonLen, const char* locale, UParseError* perror, UErrorCode* ec);
 
 
+U_CAPI USimpleNumberFormatter* U_EXPORT2
+usnumf_openForLocaleAndGroupingStrategy(
```

Comment:
How about we keep the unumf_ prefix?
For example, unumf_openSimpleForLocaleAndGroupingStrategy(...).

Reply:
Dumb question because I still don't have a good grasp on the design, but do you even need a special API for this?  Could opening a number formatting that doesn't use any of the things the new simple number formatter doesn't do just give you a simple number formatter?

Reply:
Three reasons why that's not optimal:

1. It still requires parsing the number skeleton string
2. Because the skeleton string could theoretically contain anything, we can't do tree shaking on it
3. I don't want to introduce a vtable, and adding a vtable might come at a perf cost

Reply:
>It still requires parsing the number skeleton string

Are you saying that SimpleNumberFormatter doesn't use skeleton strings, or are you just saying this is a special API that lets you bypass the skeleton string and just set a bare minimum of parameters?  If the latter, I'm on board, although that sways me in favor of Markus's naming suggestion.

>I don't want to introduce a vtable, and adding a vtable might come at a perf cost

I don't understand enough about the design of this framework (or this new functionality) to have any idea where or why this suggestion would require a viable.

Reply:
SimpleNumberFormatter doesn't use skeleton strings because it only supports a minimal number of parameters (all which don't require loading data beyond the basic number pattern).

I brought up vtable because one way to make the functions return the same type would be to introduce a vtable, but I think this is not a good idea.

Reply:
`usnum_` for all SimpleNumber and `usnumf_` for all SimpleNumberFormatter